internal/lsp/acmelsp/acmelsp.go

```diff
@@ -72,7 +93,7 @@ func PrintLocations(w io.Writer, loc []protocol.Location) error {
 		return n < 0
 	})
 	for _, l := range loc {
-		fmt.Fprintf(w, "%v\n", lsp.LocationLink(&l))
+		fmt.Fprintf(w, "%v\n%s\n", lsp.LocationLink(&l), getLine(text.ToPath(l.URI), int(l.Range.Start.Line+1)))
```

Comment:
If we add this, I think the preview should be on the same line as the location link, similar to the output of `grep`.

Reply:
Agree.